website/description_issues.qmd

```diff
@@ -135,3 +135,36 @@ The Maintainer and Author fields are automatically generated, so you don't need
 If your package uses the older approach of specifying Maintainer and Author separately in the DESCRIPTION file, it won’t be rejected for that reason alone. However, this method is discouraged, and any manual modifications that differ from the auto-generated fields will result in automatic rejection by CRAN.
 
 :::
+
+
+------------------------------------------------------------------------
+
+# References
+
+## Problem
+
+The reference link in the Description field is incorrectly formatted and isn't clickable.
+
+## Solution
+
+Remove the space after <doi:...> or <https:...> to enable the reference link.
+
+
+### Details
+
+Ideally, each package should include at least one reference in the description text to help users further explore the theory behind the package. These references should be formatted as shown in the ['tidyverse'](https://github.com/tidyverse/tidyverse/blob/main/DESCRIPTION#L8-L12) package:
```

Comment:
```suggestion
Ideally, each package should include at least one reference in the description text to help users further explore the theory behind the package. Web references should be formatted as shown in the ['tidyverse'](https://github.com/tidyverse/tidyverse/blob/main/DESCRIPTION#L8-L12) package:
```

Papers, books, etc... in this form: Authors (year) <doi:...>.
As an example take look at `Rcpp`:

> Documentation about 'Rcpp' is provided by several vignettes included in this package, via the 'Rcpp Gallery' site at <https://gallery.rcpp.org/>, the paper by Eddelbuettel and Francois (2011, <[doi:10.18637/jss.v040.i08](https://doi.org/10.18637%2Fjss.v040.i08)>), the book by Eddelbuettel (2013, <[doi:10.1007/978-1-4614-6868-4](https://doi.org/10.1007%2F978-1-4614-6868-4)>) and the paper by Eddelbuettel and Balamuta (2018, <[doi:10.1080/00031305.2017.1375990](https://doi.org/10.1080%2F00031305.2017.1375990)>); see 'citation("Rcpp")' for details.

There are examples for both, web and paper/book references. 
Maybe we should use `Rcpp` instead of `tidyverse` as example?

Reply:
Yes, that's a great example!